smithy-typescript-protocol-test-codegen/smithy-build.json

```
@@ -24,7 +24,7 @@
             },
             "license": "Apache-2.0"
           },
-          "private": true,
+          "private": true,    "generateSchemas": false,
```

nit: two different lines
```suggestion
          "private": true,
          "generateSchemas": false,
```